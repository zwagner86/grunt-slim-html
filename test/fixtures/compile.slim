doctype html
html
  head
    title Slim Examples
    meta name="keywords" content="template language"
    meta name="author" content="author"

  body
    h1 Markup examples

    #content
      p This example shows you how a basic Slim file looks like.

