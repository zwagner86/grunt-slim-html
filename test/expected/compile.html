<!DOCTYPE html><html><head><title>Slim Examples</title><meta content="template language" name="keywords" /><meta content="author" name="author" /></head><body><h1>Markup examples</h1><div id="content"><p>This example shows you how a basic Slim file looks like.</p></div></body></html>
